src/offline/component.js

```diff
@@ -53,6 +52,10 @@ function ngeoOfflineTemplateUrl($element, $attrs, ngeoOfflineTemplateUrl) {
  *     <ngeo-offline
  *       ngeo-offline-map="ctrl.map"
  *       ngeo-offline-extentsize="ctrl.offlineExtentSize">
+ *       ngeo-offline-mask-margin="::100"
+ *       ngeo-offline-min_zoom="::11"
```

Comment:
@fgravin 

I just see that here, it should be `ngeo-offline-min-zoom`, not `ngeo-offline-min_zoom`
Same for the max